pkg/volume/emptydir/empty_dir.go

```diff
@@ -514,7 +521,11 @@ func (ed *emptyDir) TearDownAt(dir string) error {
 func (ed *emptyDir) teardownDefault(dir string) error {
 	if utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolationFSQuotaMonitoring) {
 		// Remove any quota
-		err := fsquota.ClearQuota(ed.mounter, dir)
+		userNamespacesEnabled := false
+		if utilfeature.DefaultFeatureGate.Enabled(features.UserNamespacesSupport) {
+			userNamespacesEnabled = ed.pod.Spec.HostUsers != nil && !*ed.pod.Spec.HostUsers
```

Comment:
A function to capture this would be handy. The PR duplicates this functionality twice, and others may need figure out if the UserNamespaces feature is enabled.